internal/xds/translator/basicauth.go

```diff
@@ -57,13 +57,6 @@ func (*basicAuth) patchHCM(mgr *hcmv3.HttpConnectionManager, irListener *ir.HTTP
 			continue
 		}
 
-		// skip if the filter already exists
```

Comment:
why is this deleted ?

Reply:
https://github.com/envoyproxy/gateway/pull/2537#discussion_r1477468505

Reply:
not sure I understand, can you please elaborate ?

Reply:
If multiple policies target the same route, the latest wins.
If a gateway-level policy and route-level policy target the same route, the route-level policy wins.
And the per-route filter has the route name in it, so the same filter name will not appear more than once in the HCM filter chain.